scripts/build.d/zlib

```diff
@@ -2,11 +2,36 @@
 
 set -e
 
-pkgname=zlib
-pkgbranch=${VERSION:-master}
-pkgfull=${pkgname}-${pkgbranch}
+SRCSYNC=${SRCSYNC:-tarball}
 
-syncgit https://github.com/madler ${pkgname} ${pkgbranch}
+if [ "${SRCSYNC}" == "tarball" ]; then
+
+  pkgname=zlib
+  pkgver=${VERSION:-v1.2.13}
+  pkgfull=${pkgname}-${pkgver:1}
+  pkgfn=${pkgfull}.tar.gz
+  pkgurl=https://github.com/madler/zlib/archive/refs/tags/${pkgver}.tar.gz
```

Comment:
There are 2 url can be used
1. Release tarball
2. Tag tarball

Release tarball only has 1.2.13, but Tag tarball has many versions, therefore I choose Tag tarball as build script download url